src/steps/users/contact-field-equals.ts

```diff
@@ -1,81 +1,64 @@
 
-import { baseOperators } from './../../client/constants/operators';
+import { baseOperators } from '../../client/constants/operators';
 import * as util from '@run-crank/utilities';
 
 /**
  * Note: the class name here becomes this step's stepId.
- * @see BaseStep.getId()
+ @see BaseStep.getId()
  */
-export class UserFieldEqualsStep extends BaseStep implements StepInterface {
+export class ContactFieldEquals extends BaseStep implements StepInterface {
 
-  /**
-   * The name of this step: used when identifying this step to humans.
-   */
-  protected stepName: string = 'Check a field on a JSON Placeholder user';
+  protected stepName: string = 'Check a field on an Iterable Contact';
 
-  /**
-   * Type of step (either Action or Validation).
-   */
   protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
 
-  /**
-   * A string that can be evaluated as an ECMAScript-compatible regular expression. This is used to
-   * identify and evaluate this step in cucumber-like scenario files. You are encouraged to use
-   * named regex capturing groups that correspond to the expected fields below.
-   */
-  // tslint:disable-next-line:max-line-length
-  protected stepExpression: string = 'the (?<field>.+) field on JSON Placeholder user (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
+  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on iterable contact (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
 
-  /**
-   * An array of Fields that this step expects to be passed via step data. The value of "field"
-   * will be used as the field value's key when passed over the step data Struct.
-   */
   protected expectedFields: Field[] = [{
     field: 'email',
     type: FieldDefinition.Type.EMAIL,
-    description: "User's email address",
-  }, {
+    description: 'the email address of the contact',
+  },
+  {
     field: 'field',
     type: FieldDefinition.Type.STRING,
-    description: 'Field name to check',
-  }, {
+    description: 'the API name of the field',
+  },
+  {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
-  }, {
+    description: 'the operator to use when comparing expected/actual values (one of be less than, be greater than, be, contain, not be, or not contain)',
+  },
+  {
     field: 'expectedValue',
     type: FieldDefinition.Type.ANYSCALAR,
-    description: 'Expected field value',
+    description: 'field value that is expected',
   }];
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     let apiRes: any;
     const stepData: any = step.getData().toJavaScript();
     const email: string = stepData.email;
     const field: string = stepData.field;
+    const operator: string = stepData.operator;
```

Comment:
@saludangelito is this `toLowerCase()`'d in the `util.compare()`?

Reply:
No, but upon check when inputting a valid but different cased operator, it returns an unclear error. I should fix that.

Reply:
@saludangelito re-upping this comment, as I don't think it was addressed in your most recent changes.

Reply:
My fix for this one is actually made in the util instead since i noticed most of the cogs don't lowercase the operator. I will be pushing the fix along with the fix on the bug on running the unit test since i am still having problems running jest config(which runs the tests) along with npm run build. Though running the jest config alone works.